google/cloud/networkconnectivity/README.md

```diff
@@ -0,0 +1,93 @@
+provider to your Virtual Private Cloud (VPC) network.
+
+This library is **experimental**. Its APIs are subject to change without notice.
+
+Please note that the Google Cloud C++ client libraries do **not** follow
+[Semantic Versioning](https://semver.org/).
+
+## Supported Platforms
+
+- Windows, macOS, Linux
+- C++14 (and higher) compilers (we test with GCC >= 7.3, Clang >= 6.0, and
+  MSVC >= 2017)
+- Environments with or without exceptions
+- Bazel (>= 4.0) and CMake (>= 3.5) builds
+
+## Documentation
+
+- Official documentation about the [Network Connectivity API][cloud-service-docs] service
+- [Reference doxygen documentation][doxygen-link] for each release of this
+  client library
+- Detailed header comments in our [public `.h`][source-link] files
+
+## Quickstart
+
+The [quickstart/](quickstart/README.md) directory contains a minimal environment
+to get started using this client library in a larger project. The following
+"Hello World" program is used in this quickstart, and should give you a taste of
+this library.
+
+<!-- inject-quickstart-start -->
+
+```cc
+#include "google/cloud/networkconnectivity/hub_client.h"
+#include <iostream>
+#include <stdexcept>
+
+int main(int argc, char* argv[]) try {
+  if (argc != 2) {
+    std::cerr << "Usage: " << argv[0] << " project-id\n";
+    return 1;
+  }
+
+  namespace networkconnectivity = ::google::cloud::networkconnectivity;
+  auto client = networkconnectivity::HubServiceClient(
+      networkconnectivity::MakeHubServiceConnection());
+
+  auto const parent = std::string{"projects/"} + argv[1] + "/locations/global";
+  for (auto h : client.ListHubs(parent)) {
+    if (!h) throw std::runtime_error(h.status().message());
+    std::cout << h->DebugString() << "\n";
+  }
+
+  return 0;
+} catch (std::exception const& ex) {
+  std::cerr << "Standard exception raised: " << ex.what() << "\n";
+  return 1;
+}
+```
+
+<!-- inject-quickstart-end -->
+
+- Packaging maintainers or developers who prefer to install the library in a
+  fixed directory (such as `/usr/local` or `/opt`) should consult the
+  [packaging guide](/doc/packaging.md).
+- Developers wanting to use the libraries as part of a larger CMake or Bazel
+  project should consult the [quickstart guides](#quickstart) for the library
+  or libraries they want to use.
+- Developers wanting to compile the library just to run some of the examples or
+  tests should read the current document.
+- Contributors and developers to `google-cloud-cpp` should consult the guide to
+  [setup a development workstation][howto-setup-dev-workstation].
+
+## Contributing changes
+
+See [`CONTRIBUTING.md`](/CONTRIBUTING.md) for details on how to
+contribute to this project, including how to build and test your changes
+as well as how to properly format your code.
+
+## Licensing
+
+Apache 2.0; see [`LICENSE`](/LICENSE) for details.
+
+[cloud-service-docs]: https://cloud.google.com/network-connectivity/docs/concepts
```

Comment:
I would link to `https://cloud.google.com/network-connectivity/docs` and let the redirect do its thing.

Reply:
Done.